test/src/engine/BMGameTest.php

```diff
@@ -669,12 +665,13 @@ public function test_do_next_step_specify_dice() {
         $recipe2 = '(6) (12) (20) (20)';
         $button1->load($recipe1);
         $button2->load($recipe2);
-        $this->object->activeDieArrayArray = array($button1->dieArray,
-                                                   $button2->dieArray);
-        $this->object->swingRequestArrayArray = array(array(), array());
-        $this->object->gameScoreArrayArray = array_fill(0, $this->object->nPlayers, array(0, 0, 0));
+        $player = $this->object->playerArray[0];
+        $player->activeDieArray = $button1->dieArray;
+        $player = $this->object->playerArray[1];
+        $player->activeDieArray = $button2->dieArray;
```

Comment:
This is pretty nit-level for a test, but make these `$player1` and `$player2` to match `$button1` and `$button2`.